tutorials/commands/job-submit.rst

```diff
@@ -0,0 +1,55 @@
+.. _job-submit:
+
+==========================
+How to Submit Jobs in Flux
+==========================
+
+Do you want to submit a job to Flux? Here's a short tutorial on how to do so!
+
+----------------------------
+Submit Jobs Using Flux's CLI
+----------------------------
+
+From within a Flux instance, you can submit your job on the command line with
+optional arguments, resource options, per task options, and per resource
+options:
+
+.. code-block:: sh
+
+    $ flux mini submit --nodes=2 --ntasks=4 --cores-per-task=2 ./my_compute_script.lua 120
```

Comment:
was thinking if there are any other important options to mention, the only one I could think of is `--queue`.

IMO, that's more important than `--cores-per-task`.  You could probably just mention that there are many advanced ways to request resources besides `--nodes` and `--ntasks`, and to see the manpage for details.

Reply:
If you are open to it, I like having (toward the end) a big block of just examples with description, that sort of show all the options that a flux command can provide. E.g., I think I linked this before, but this example comes to mind! https://rse-ops.github.io/knowledge/docs/schedulers/slurm.html#command-quick-reference. Sometimes people's eyes will glaze over the text and they just want to find the right thing to copy paste :) (guilty!)

Reply:
oh that's a good idea